rust/src/libp2p_bridge.rs

```diff
@@ -37,13 +37,21 @@ pub fn create_and_run_network(network_id: u32, zig_handler: u64, self_port: i32,
 /// The caller must ensure that `message_str` points to valid memory of `message_len` bytes.
 #[no_mangle]
 #[allow(clippy::missing_safety_doc)]
-pub unsafe fn publish_msg_to_rust_bridge(network_id:u32, _topic_id: u32, message_str: *const u8, message_len: usize){
+pub unsafe fn publish_msg_to_rust_bridge(network_id:u32,topic_id: u32, message_str: *const u8, message_len: usize){
         let message_slice = std::slice::from_raw_parts(message_str, message_len);
         println!("rustbridge-{network_id}:: publishing message s={:?}",message_slice);
         let message_data = message_slice.to_vec();
 
         // TODO: get the topic mapping from topic_id
-        let topic = gossipsub::IdentTopic::new("block");
+        let topic = match topic_id {
+            0 => gossipsub::IdentTopic::new("block"),
+            1 => gossipsub::IdentTopic::new("vote"),
+            unknown_id => {
+                println!("Invalid topic_id: {unknown_id}");
+                return;
+            },
+        };
```

Comment:
this is more something that we will need to do during the cleanup than in this PR, but I suggest that we reduce our reliance on rust by doing this in zig and passing the topic/string as a parameter. Less surface to maintain in rust. But it's ok for now.